fastlane/Fastfile

```diff
@@ -0,0 +1,119 @@
+# Change the syntax highlighting to Ruby
+
+# All lines starting with a # are ignored when running `fastlane`
+
+# This is the minimum version number required. 
+# Update this, if you use features of a newer version
+fastlane_version "0.4.2" 
+
+lane :build_for_test do
+  xcodebuild(
+    scheme: 'Kiosk',
+    workspace: 'Kiosk.xcworkspace',
+    destination: "OS=8.1,name=iPad Air",
+  )
+end
+
+lane :test do
+  xctest(
+    destination: "OS=8.1,name=iPad Air",
+  )
+end
+
+lane :oss do
+  keys = ['ArtsyAPIClientSecret', 
+    'ArtsyAPIClientKey', 
+    'HockeyProductionSecret', 
+    'HockeyBetaSecret', 
+    'MixpanelProductionAPIClientKey', 
+    'MixpanelStagingAPIClientKey', 
+    'CardflightAPIClientKey', 
+    'CardflightAPIStagingClientKey', 
+    'CardflightMerchantAccountToken', 
+    'CardflightMerchantAccountStagingToken', 
+    'BalancedMarketplaceToken', 
+    'BalancedMarketplaceStagingToken']
+
+    commands = keys.map { |key|
+      command = "bundle exec pod keys set #{key} '-'"
+      if key == keys.first
+        command += " Eidolon"
+      end
+
+      command
+    }.join(' ; ')
+
+    sh "cd .. ; #{commands} ; pod install"
+end
+
+lane :build_for_deploy do
+  xcodebuild(
+    scheme: 'Kiosk',
+    workspace: 'Kiosk.xcworkspace',
+    sdk: "iphoneos",
+    configuration: "release",
+  )
+end
+
+lane :deploy do 
+
+  # Increment build number to current date
+  build_number = Time.new.strftime("%Y.%m.%d")
+  increment_build_number({
+    build_number: build_number,
+    xcodeproj: './Kiosk.xcodeproj',
+  })
+
+  # Set version number to the one at the top of the CHANGELOG
+  readme = File.read("../docs/CHANGELOG.md")
+  latest_version = readme.split("\n## ").first
+  first_line = latest_version.split("\n").first
+  version_number = first_line.split(" ")[1]
+
+  increment_version_number(
+    release_task: version_number,
+    xcodeproj: './Kiosk.xcodeproj',
+  )
+
+  # Generate release notes from CHANGELOG
+  release_notes = latest_version.split("\n")[1..-1].join("\n")
+
+  # Build
+  ipa({
+    workspace: "Kiosk.xcworkspace",
+    configuration: "Release",
+    scheme: "Kiosk",
+    destination: "build",
+    embed: "fastlane/Distribution_net.artsy.kiosk.beta.mobileprovision",
+    identity: "iPhone Distribution: ART SY INC",
+    'no-clean' => ' ',
+  })
+
+  # Push to Hockey
+  hockey({
+    api_token: ENV['HOCKEY_API_TOKEN'],
+    notes_type: 1, # Specifies notes are in markdown format
+    notes: release_notes,
+  })
+
+  # Post message to slack
+  slack(
+    message: "There is a new version of the Kiosk app available. Download it at http://artsy.net/kioskbeta",
+    success: true,        # optional, defaults to true
+    payload: {            # optional, lets you specify any number of your own Slack attachments
+      'Version' => version_number,
+      'What\'s new' => release_notes,
+    },
+    default_payloads: [],
+  )
+
+  # Tag release and push to GitHub
+  sh "cd .. ; git tag #{version_number} ; git push --tags"
```

Comment:
Yes, you can use [push_to_git_remote](https://github.com/KrauseFx/fastlane/blob/master/Actions.md#push_to_git_remote). I haven't used it myself, but I heard good things about it :wink: